MANIFEST.in

```diff
@@ -7,10 +7,13 @@ exclude .installed.cfg
 include *.yml
 exclude *.yml
 
+include *.yaml
+exclude *.yaml
```

Comment:
(what effect does first `include` then `exclude` have?)

Reply:
`include` makes `check-manifest` to not complain about a file under version control but not in the sdist.
`exclude` does not include this file in the sdist as the configuration files are not needed there.

I know, it looks ugly but at least it works until I find a better solution.